inst/extdata/saga_module_depends.csv

```diff
@@ -0,0 +1,47 @@
+order,module,dep1,dep2,tile_artifact
+1,sinksfilled,,,
+2,sinkroute,,,
+3,dem_preproc,sinkroute,,
+4,slope_aspect_curve,,,
+5,tcatchment,,,
+6,tca,,,
+7,scatchment,sinksfilled,tcatchment,
+8,twi,slope_aspect_curve,scatchment,
+9,channelsnetwork,sinksfilled,tcatchment,
+10,overlandflow,sinksfilled,channelsnetwork,
+11,overlandflow2,sinksfilled,channelsnetwork,
+12,multiresflatness,,,TRUE
+13,multiresflatness2,,,TRUE
+14,multiresflatness3,,,TRUE
+15,tri,,,
+16,convergence,,,
+17,openness,,,
+18,dah,,,
+19,tpi,,,
+20,ridgevalley,sinksfilled,,
+21,mrn,sinksfilled,,
+22,flowaccumulation,sinksfilled,,TRUE
+23,flowaccumulation,sinksfilled,,TRUE
```

Comment:
@gcperk are these rows meant to be duplicated?

Reply:
Yes - I deleted one row